Makefile.am

```diff
@@ -116,9 +116,22 @@ test:
 	bin/urweb -boot -noEmacs -dbms sqlite -db $(TESTDB) -demo /Demo demo
 	rm -f $(TESTDB)
 	sqlite3 $(TESTDB) < demo/demo.sql
-	demo/demo.exe & echo $$! > $(TESTPID)
+	demo/demo.exe -a 127.0.0.1 & echo $$! > $(TESTPID)
```

Comment:
You mean we should keep the original one and add my change as additional test?
I think then I would like to reorganize the code a little; having almost the same code thrice is a little too much for my taste...